scripts/i.oif/testsuite/test_i_oif.py

```diff
@@ -0,0 +1,175 @@
+import grass.script as gs
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+import os
+from pathlib import Path
```

Comment:
If there was a potential improvement, for new files having the imports sorted with isort would be great. It will make less changes to apply when we finally enforce it.